Modules/_cursesmodule.c

```diff
@@ -2596,27 +2624,41 @@ which will be between 0 (no component) and 1000 (maximum amount of component).
 [clinic start generated code]*/
 
 static PyObject *
-_curses_color_content_impl(PyObject *module, short color_number)
-/*[clinic end generated code: output=cb15cf3120d4bfc1 input=5555abb1c11e11b7]*/
+_curses_color_content_impl(PyObject *module, int color_number)
+/*[clinic end generated code: output=17b466df7054e0de input=badb7d68ffbb0e93]*/
 {
-    short r,g,b;
+    PyObject *return_value = NULL;
+    _NCURSES_COLOR_VAL_TYPE r,g,b;
 
     PyCursesInitialised;
     PyCursesInitialisedColor;
 
-    if (color_content(color_number, &r, &g, &b) != ERR)
-        return Py_BuildValue("(iii)", r, g, b);
-    else {
+    if (color_number < _NCURSES_COLOR_VAL_MIN) {
+        PyErr_SetString(PyExc_OverflowError,
+                        "signed " _NCURSES_COLOR_VAL_TYPE_STR " is less than minimum");
```

Comment:
Please limit lines to 79 chars (https://www.python.org/dev/peps/pep-0007/#code-lay-out).

For what it's worth, I think the error messages added in this PR are too vague.

Reply:
Fixed the formatting and made the error messages a bit more specific.